docs/app/services/version/version.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable, Output, EventEmitter } from '@angular/core';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
+
+@Injectable()
+export class VersionService {
+
+    version: BehaviorSubject<Version> = new BehaviorSubject<Version>(Version.Angular);
+
+    constructor() {
+        this.toggle(this.toVersion(window.localStorage.getItem('version')));
+    }
+
+    toggle(version: Version): void {
```

Comment:
Rename this to `setVersion` as `toggle` implies it switches to the next version which isn't the behavior of the method.